cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -2069,7 +2069,7 @@ public static final class Metadata {
     public static final String STORAGE_PROVIDER_NOSQL = "nosql";
     public static final String STORAGE_PROVIDER_ELASTICSEARCH = "elastic";
 
-    public static final String METADATA_WRITER_SUBSCRIBER = "metadata.writer";
```

Comment:
Could you please leave this constant and use it in MetadataSubscriberService ?

Reply:
done.